.env

```diff
@@ -3,3 +3,6 @@ GROBID_TAG=0.5.4
 REVISION=develop
 # $GROBID_TAG-$REVISION
 IMAGE_TAG=0.5.4-develop
+
+# all grobid tags built by ci
+ALL_GROBID_TAGS=0.5.3:0.5.4
```

Comment:
`:` is a valid character in the Docker image `name:tag` syntax, so it may create confusion; `,` seems more standard for a list?